endaq/calc/utils.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import annotations
+
+import typing
+from typing import Optional
+import warnings
+
+import numpy as np
+import pandas as pd
+
+
+def sample_spacing(
+    df: pd.DataFrame, convert: typing.Literal[None, "to_seconds"] = "to_seconds"
+):
```

Comment:
Should add a return type hint

Reply:
see 06f2b0ae993894190bbe6c5aa510cddf75c58bde